one-file-tasks/Classes-ex1.py

```diff
@@ -0,0 +1,19 @@
+class Vehicle:
+    def __init__(self, name, max_speed, mileage):
+        self.name = name
+        self.max_speed = max_speed
+        self.mileage = mileage
+
+    def seating_capacity(self, capacity):
+        return f"The seating capacity of a {self.name} is {capacity} passengers. And is {self.size}"
+
+class Bus(Vehicle):
+    def __init__(self,name,max_speed, milage, size='big'):
+        super().__init__(name, max_speed, milage)
+        self.size=size
+    
+    def seating_capacity(self, capacity):
```

Comment:
What's the point of this method? You have it already defined in the parent class and you can use it without defining it again in the child class.